gcloud/datastore/entity.py

```diff
@@ -104,55 +102,6 @@ def exclude_from_indexes(self):
         """
         return frozenset(self._exclude_from_indexes)
 
-    @property
-    def _must_key(self):
```

Comment:
So now if an Entity has no key, we rely on the backend to fail during the save? Or is there some other code path I'm not thinking of.

Also, but doing this we resolve #471